source/d3d8to9_device.cpp

```diff
@@ -775,12 +792,7 @@ HRESULT STDMETHODCALLTYPE Direct3DDevice8::EndStateBlock(DWORD *pToken)
 	if (pToken == nullptr)
 		return D3DERR_INVALIDCALL;
 
-	HRESULT hr = ProxyInterface->EndStateBlock(reinterpret_cast<IDirect3DStateBlock9 **>(pToken));
```

Comment:
Please consider keeping this in place, as per above.

Reply:
Ok, good catch.  I had not realized that state blocks can be added that way.  Fixed!